server/helper_test.go

```diff
@@ -449,6 +461,39 @@ func (th *testHelper) assertEphemeralMessage(t *testing.T, userID, channelID, me
 	}
 }
 
+func (th *testHelper) assertNoEphemeralMessage(t *testing.T, userID, channelID, message string, maxWaitTime time.Duration) {
+	t.Helper()
+
+	websocketClient := th.GetWebsocketClientForUser(t, userID)
+
+	for {
+		select {
+		case event, ok := <-websocketClient.EventChannel:
+			if !ok {
+				t.Fatal("channel closed before getting websocket event for ephemeral message")
+			}
+
+			if event.EventType() == model.WebsocketEventEphemeralMessage {
+				data := event.GetData()
+				postJSON, ok := data["post"].(string)
+				require.True(t, ok, "failed to find post in ephemeral message websocket event")
+
+				var post model.Post
+				err := json.Unmarshal([]byte(postJSON), &post)
+				require.NoError(t, err)
+
+				if post.ChannelId == channelID && post.Message == message {
+					t.Fatalf("received undesired ephemeral message in channel %s: %s", post.ChannelId, post.Message)
+				}
+			}
```

Comment:
I wonder if asserting "no" message should be simpler and just accept the `channelID`? I could see case where we start emitting a different message but don't check for that because of this equality test.

Reply:
I intended to test specifically that this message (message as in content, not post) was not sent. I guess the name of the method is not clear enough, I wish we talked about ephemeralPost, then the word 'message' would be more clear :P 

Will rename to `assertNoEphemeralMessageWithContent`